site/content/docs/5.2/getting-started/vite.md

```diff
@@ -0,0 +1,197 @@
+
+   ```sh
+   npm i --save bootstrap @popperjs/core
+   ```
+4. **Install additional dependency.** In addition to Vite and Bootstrap, we need another dependency (Sass) to properly import and bundle Bootstrap's CSS.
+
+   ```sh
+   npm i --save-dev sass
+   ```
+
+Now that we have all the necessary dependencies installed and setup, we can get to work creating the project files and importing Bootstrap.
+
+## Project structure
+
+We've already created the `my-project` folder and initialized npm. Now we'll also create our `src` folder, stylesheet, and JavaScript file to round out the project structure. Run the following from `my-project`, or manually create the folder and file structure shown below.
+
+```sh
+mkdir {src,src/js,src/scss}
+touch src/index.html src/js/main.js src/scss/styles.scss vite.config.js
+```
+
+When you're done, your complete project should look like this:
+
+```text
+my-project/
+├── src/
+│   ├── js/
+│   │   └── main.js
+│   └── scss/
+│   |   └── styles.scss
+|   └── index.html
+├── package-lock.json
+├── package.json
+└── vite.config.js
+```
+
+At this point, everything is in the right place, but Vite won't work because we haven't filled in our `vite.config.js` yet.
+
+## Configure Vite
+
+With dependencies installed and our project folder ready for us to start coding, we can now configure Vite and run our project locally.
+
+1. **Open `vite.config.js` in your editor.** Since it's blank, we'll need to add some boilerplate config to it so we can start our server. This part of the config tells Vite were to look for our project's JavaScript and how the development server should behave (pulling from the `src` folder with hot reload).
+
+   <!-- eslint-skip -->
+   ```js
+   const path = require('path')
+
+   export default {
+     root: path.resolve(__dirname, 'src'),
+     server: {
+       port: 8080,
+       hot: true
+     }
+   }
+   ```
+
+2. **Next we fill in `src/index.html`.** This is the HTML page Vite will load in the browser to utilize the bundled CSS and JS we'll add to it in later steps.
+
+   ```html
+   <!doctype html>
+   <html lang="en">
+     <head>
+       <meta charset="utf-8">
+       <meta name="viewport" content="width=device-width, initial-scale=1">
+       <title>Bootstrap w/ Vite</title>
+     </head>
+     <body>
+       <div class="container py-4 px-3 mx-auto">
+         <h1>Hello, Bootstrap and Vite!</h1>
+         <button class="btn btn-primary">Primary button</button>
+       </div>
+       <script type="module" src="./js/main.js"></script>
+     </body>
+   </html>
+   ```
+
+   We're including a little bit of Bootstrap styling here with the `div class="container"` and `<button>` so that we see when Bootstrap's CSS is loaded by Vite.
+
+3. **Now we need an npm script to run Vite.** Open `package.json` and add the `start` script shown below (you should already have the test script). We'll use this script to start our local Vite dev server.
+
+   ```json
```

Comment:
```suggestion
   ```jsonc
```

Reply:
Indeed it would be the right format to use but unfortunately [Chroma doesn't handle JSON-C](https://github.com/alecthomas/chroma#supported-languages) so the syntax wouldn't be highlighted in the documentation
![Screenshot from 2022-05-24 08-54-08](https://user-images.githubusercontent.com/17381666/169967538-abf58e5a-f420-4b82-b0e7-e0d7b320e20d.png)
instead of what we have [with `json` right now](https://deploy-preview-36412--twbs-bootstrap.netlify.app/docs/5.2/getting-started/vite/#configure-vite) which is rendered like this:
![Screenshot from 2022-05-24 08-56-30](https://user-images.githubusercontent.com/17381666/169967961-f03fcd88-e0dd-434b-96ca-334449fa18e3.png)

Reply:
This doesn't have to do with browsers, though.

This has to do with Hugo (which is using chroma) and GitHub itself.

Are you saying Hugo doesn't highlight this if we use `jsonc`? If so, I'd remove the comments altogether.

Reply:
No it doesn't have to do with browsers. I've just displayed the rendering to show the difference. Hugo doesn't highlight it because JSON-C is not handled/recognized by Chroma.

Reply:
I see there's an issue https://github.com/alecthomas/chroma/issues/629 so it might get implemented at some point.

That being said, maybe it's better to just remove the comments from json.

Reply:
Even if comments doesn't exist in JSON format, IMHO they bring something here to the user who can understand at a glance that this part is not a complete file but only an extract (even if it is rather obvious).
I'd vote to keep them and replace `json` by `jsonc` when it will be available.
But I don't have neither a strong opinion on that, if you really think that's an issue, I can remove them in the 3 guides.
@mdo would you prefer to keep the comments or not?